docs/docs/basics/serving-static-files.md

```diff
@@ -33,3 +37,25 @@ In production, only files that are in the `/public` directory at build time will
 :::caution
 Be sure not to have a static file with the same name as a file in the `/routes` directory as this will result in a conflict.
 :::
+
+## Using a Custom Directory ✨
+
+Even though Dart Frog uses the `public` directory for serving static files by default, you can also specify a custom directory by creating a [custom entrypoint](/docs/advanced/custom_entrypoint).
+
+Create a `main.dart` at the root of your project with the following contents:
```

Comment:
Shouldn't this be in the singular?

```suggestion
Create a `main.dart` at the root of your project with the following content:
```

Reply:
I think technically either is fine but looks like file contents is more commonly use to refer to this
https://english.stackexchange.com/questions/56831/file-content-vs-file-contents

Reply:
Interesting! Thanks for explanation